contracts/scripts/utils.js

```diff
@@ -31,110 +31,93 @@ async function main() {
   console.log('Token address:', token.address);
 }
 
-async function cancelTransaction(nonce) {
+// Helpers
+
+/**
+ * @param number nonce
+ * @param string gasPrice
+ * If a transaction is stuck, you can use this function to cancel that transaction as long as the gasPrice value is larger
```

Comment:
nit: we should probly stick to the common format where first is the func summary, then params

Reply:
great catch ty!